django/db/models/fields/__init__.py

```diff
@@ -843,18 +866,12 @@ def get_flatchoices(self, include_blank=True,
         first_choice = blank_choice if include_blank else []
         return first_choice + list(self.flatchoices)
 
-    def _get_val_from_obj(self, obj):
```

Comment:
Not sure we can remove this method or its usage without thinking about backwards compatibility. It's documented in docs/howto/custom-model-fields.txt